helix-view/Cargo.toml

```diff
@@ -28,3 +28,4 @@ slotmap = "1"
 
 serde = { version = "1.0", features = ["derive"] }
 toml = "0.5"
+log = "~0.4"
```

Comment:
Sidenote: @pickfire should we simply re-export `log` from `helix_core`?

Reply:
I don't think it's necessary. We can just do this, it seemed better. For log it's easy to keep them in sync.